README.md

```diff
@@ -203,6 +203,7 @@ A collection of android security related resources.
 
 ### Vulnerable Applications for practice
 
+1. [Oversecured Vulnerable Android App (OVAA)](https://github.com/oversecured/ovaa)
```

Comment:
Move this to the bottom of the list (above all dead strike-through entries)

Reply:
Done!